drag_and_drop_v2/public/css/drag_and_drop.css

```diff
@@ -83,8 +83,14 @@
     transform: translate(-50%, -50%); /* These blocks are to be centered on their absolute x,y position */
 }
 
+/* Focused option */
+.xblock--drag-and-drop .drag-container .item-bank .option:focus,
+.xblock--drag-and-drop .drag-container .item-bank .option:hover {
+    transform: scale(1.1);
```

Comment:
@itsjeyd You could also just put:

```
.xblock--drag-and-drop .drag-container .item-bank .option {
    outline-color: white;
    outline-style: none;
}
.xblock--drag-and-drop .drag-container .item-bank .option:hover,
.xblock--drag-and-drop .drag-container .item-bank .option:focus {
    outline-width: 2px;
    outline-style: solid;
}
```

And then on the element itself in the HTML, you can use an inline `style="outline-color: {{textColor}}"` so that the outline will match the text color, which should always be a contrasting color.